src/solver/conv_direct_naive_conv.cpp

```diff
@@ -100,6 +102,23 @@ bool IsOutputInt32(const ProblemDescription& problem)
            problem.out_data_type == miopenInt32;
 }
 
+/*
+ * These return strings are synced with the kernel source file
+ */
+std::string TypeToString(miopenDataType_t data_type)
+{
+    if(data_type == miopenFloat)
+        return "float";
+    if(data_type == miopenHalf)
+        return "half";
+    if(data_type == miopenFloat8)
+        return "float8";
+    if(data_type == miopenBFloat8)
+        return "bfloat8";
+    if(data_type == miopenDouble)
+        return "double";
+    MIOPEN_THROW("Unimplemented type in FP8 kernels");
+}
```

Comment:
This function exists elsewhere in MIOpen please avoid duplicate code.

Reply:
This function is indeed a duplicate of 
inline std::string GetDataType(miopenDataType_t type)
in
miopen/datatype.hpp